pim-tool-back/src/main/java/vn/elca/training/model/entity/ProjectMember.java

```diff
@@ -0,0 +1,65 @@
+package vn.elca.training.model.entity;
+
+import javax.persistence.*;
+
+@Entity
+@Table(name = "PROJECT_MEMBER_TABLE")
+public class ProjectMember {
```

Comment:
This solution is OK.

However, entity "ProjectMember" is the parent in this relationship. And most of the cases, we would want "Project" to be the parent and then access its members from entity "Project". You can try to use EmbeddedId to achieve this.

https://www.baeldung.com/spring-jpa-embedded-method-parameters